docroot/themes/humsci/humsci_basic/src/scss/elements/_base.scss

```diff
@@ -0,0 +1,48 @@
+html {
+  font-size: 16px;
+
+  @include colorful {
+    font-family: $colorful-font--sans;
+  }
+}
+
+h1 {
```

Comment:
All of the elements styled in this page (with the exception of `html` and `body`) should include the `:not([class])` rule. This will prevent applying a default styles that will then have to be overridden when applying classes.

Reply:
While were at it we should also add a `/components/_patterns.typography.scss` that adds classes for all of these element styles:
```
.cmp-heading-1 {
  @include hb_heading1;
}
```

This will make it way easier to override default styles (ie. `<h2 class="cmp-heading-h3"></h2>`).

Reply:
I added a utility file!